node-it/src/test/scala/com/wavesplatform/it/account/storage/RemoveEntrySuite.scala

```diff
@@ -0,0 +1,192 @@
+package com.wavesplatform.it.account.storage
+
+import com.wavesplatform.account.KeyPair
+import com.wavesplatform.common.utils.{Base58, EitherExt2}
+import com.wavesplatform.it.api.SyncHttpApi._
+import com.wavesplatform.it.sync._
+import com.wavesplatform.it.util._
+import com.wavesplatform.lang.v1.compiler.Terms.{CONST_BOOLEAN, CONST_LONG, CONST_STRING}
+import com.wavesplatform.lang.v1.estimator.v2.ScriptEstimatorV2
+import com.wavesplatform.transaction.smart.SetScriptTransaction
+import com.wavesplatform.transaction.smart.script.ScriptCompiler
+
+class RemoveEntrySuite extends BaseSuite {
+
+  val writeEntry =
+    """
+      |@Callable(i)
+      |func write%f(k: String, v: %t) = {
+      |  [%fEntry(k, v)]
+      |}
+      |""".stripMargin
+
+  val writeEntriesFunc =
+    """
+      |@Callable(i)
+      |func write%sEntries() = {
+      |    FOLD<%s>(a, [], writeEntry)
```

Comment:
you can shorten such functions to one line
`func write%sEntries() = FOLD<%s>(a, [], writeEntry)`

Reply:
ok